examples/cra-kitchen-sink/src/stories/index.js

```diff
@@ -90,7 +90,7 @@
 //   .add('with some emoji', () => (
```

Comment:
what is this file needed for ?

Reply:
Looks like @danielduan forgot to remove it